src/main/java/ru/practicum/shareit/exception/ErrorHandler.java

```diff
@@ -0,0 +1,32 @@
+package ru.practicum.shareit.exception;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+@Slf4j
+public class ErrorHandler {
+    @ExceptionHandler
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public Exception incorrectValue(final IncorrectInputException e) {
+        log.warn("Error 400: " + e.getMessage());
```

Comment:
При логировании параметров, лучше использовать заполнитель(placeholder) `{}`, а не конкатенацию. 
Тогда код будет выглядить более лаконично:

```suggestion
        log.warn("Error 400: {}", e.getMessage());
```

Reply:
Исправлено.